arduino-ide-extension/src/browser/contributions/help.ts

```diff
@@ -120,6 +124,10 @@ export class Help extends Contribution {
       commandId: IDEUpdaterCommands.CHECK_FOR_UPDATES.id,
       order: '7',
     });
+    registry.registerMenuAction(ArduinoMenus.HELP__FIND_GROUP, {
+      commandId: Help.Commands.PRIVACY_POLICY.id,
+      order: '8',
```

Comment:
I suggest switching places between the **Help > Privacy Policy** and **Help > Check for Arduino IDE updates** menu items.

It seems more logical to me to have all the website visit menu items grouped together, rather than having the very different **Check for Arduino IDE updates** menu item interspersed.

Before:

- Help
  - Getting Started
  - Environment
  - Troubleshooting
  - Reference
  - Find in Reference
  - Frequently Asked Questions
  - Visit Arduino.cc
  - **Check for Arduino IDE Updates**
  - Privacy Policy
  - About Arduino IDE
  - Toggle Developer Tools

After:

- Help
  - Getting Started
  - Environment
  - Troubleshooting
  - Reference
  - Find in Reference
  - Frequently Asked Questions
  - Visit Arduino.cc
  - Privacy Policy
  - **Check for Arduino IDE Updates**
  - About Arduino IDE
  - Toggle Developer Tools

Reply:
> Should the URL be an internationalized string (`nls.localize('https://www.arduino.cc/en/privacy-policy')`) so that it can be pointed at the appropriate page (e.g., https://www.arduino.cc/it/privacy-policy)?
> 
> I tried changing my browser locale to `it` in order to see whether arduino.cc automatically redirects https://www.arduino.cc/en/privacy-policy to https://www.arduino.cc/it/privacy-policy, and it didn't, but I don't know much about the subject.

@per1234 I thought about that too, but seeing the page was in English-only I didn't localize the URL. @ubidefeo, do you think we should ask someone to translate that page?

Reply:
@AlbyIanna there is an Italian language version of the page:

https://www.arduino.cc/it/privacy-policy

I don't think there are any others at the moment.

Reply:
I swear it didn't work the first time I tried that URL 😅  It just redirected me to the English version of the page. Weird.

Anyway, should we localize it? I don't know, I think it'd be kinda strange if we'd localize it only for Italian folks.

Reply:
I don't personally find it strange, but I also don't have strong feelings about the subject either way. I added the above only as a [comment](https://github.com/arduino/arduino-ide/commit/d3e5dceac017cf9a6b11ccb8d1f3df78babd968f#r67904231), and not as part of my review for that reason.

I'll go ahead and submit an approving review now since the review suggestion I made has been resolved (thanks!) and we are on the same page regarding the existence of an Italian version of the page.

I'll leave the decision on whether to internationalize that URL string to you or any other interested parties.